impl/maven-impl/src/main/java/org/apache/maven/impl/DefaultDependencyResolver.java

```diff
@@ -126,7 +141,11 @@ public DependencyResolverResult collect(@Nonnull DependencyResolverRequest reque
                 final CollectResult result =
                         session.getRepositorySystem().collectDependencies(systemSession, collectRequest);
                 return new DefaultDependencyResolverResult(
-                        null, null, result.getExceptions(), session.getNode(result.getRoot(), request.getVerbose()), 0);
+                        null,
+                        moduleCache(),
```

Comment:
as its dry, and static, inline. Ide shoud give warning on method or constructor that value is always same.

Reply:
I'm not sure to understand. Inline the call to `moduleCache()`? This is uneasy, as that method does lazy instantiation.

Reply:
sorry wrong assumption. This is called every time as this the only return therefore default.